cypress/e2e/buy-crypto-modal.test.ts

```diff
@@ -0,0 +1,36 @@
+import { getTestSelector } from '../utils'
+
+describe('Buy Crypto Modal', () => {
+  it('should open and close', () => {
+    cy.visit('/')
+
+    cy.get(getTestSelector('buy-fiat-button'))
+      .click()
+      .then(() => {
+        cy.get(getTestSelector('fiat-onramp-modal')).should('be.visible')
+      })
```

Comment:
Can this be done with less nesting? eg would it perform the same if expressed as

    cy.get(getTestSelector('buy-fiat-button')).click()
    cy.get(getTestSelector('fiat-onramp-modal')).should('be.visible')
    
If so, I think the latter should be preferred, as it makes it much easier to read without having to keep scoping/nesting/closures in mind.